config.m4

```diff
@@ -181,6 +181,12 @@ if test "$PHP_MONGODB" != "no"; then
                                [MongoDB: Use system libmongoc [default=no]])],
                [no],
                [no])
+  PHP_ARG_WITH([mongodb-client-side-encryption],
+                 [whether to enable client-side encryption],
+                 [AS_HELP_STRING([--with-mongodb-client-side-encryption=@<:@auto/yes/no@:>@],
+                                 [MongoDB: Enable client-side encryption [default=auto]])],
+                 [auto],
+                 [no])
```

Comment:
These five can be re-aligned before merging. I forgot to do this when switching to `PHP_ARG_WITH`.